### ISSUE TITLE
Explicit indication of how to override the go.sh boot process

### ISSUE THREAD

**Issue by markfinger** [ic-labs/django-icekit#143]
I tried pdb'ing in the settings file and the boot process locks up because the migrate check pipes all output to a file.

Tai suggested a work around: `./go.sh bash.sh`. This prevents the django setup/bootstrap process from occurring and provides the ability to manually introspect `manage.py`.

While this is an easy work around, it's a hidden feature that Tai suggested should be indicated during the `go.sh` boot.

cc: @mrmachine

**Comment by markfinger** [comment]
@mrmachine would this be as simple as something like `echo "Run ./go.sh bash.sh to avoid automated setup"`? If so, I'm happy to make a PR

**Comment by mrmachine** [comment]
Yep. Probably here https://github.com/ic-labs/django-icekit/blob/518955743f58183cb25ed7efc76444130405b8e8/icekit/bin/setup-django.sh#L6-L8 add something like "To avoid running `setup-django.sh` automatically, run `docker-compose run ... django bash.sh` or `./go.sh bash.sh`", with a blank line above/after and maybe indented or something to make it stand out a bit more. Or maybe write to stderr, or otherwise colorize it or something.
